packages/lesswrong/lib/editor/ellipsize.jsx

```diff
@@ -37,7 +37,7 @@ export const commentExcerptFromHTML = (html, truncationCharCount) => {
   const htmlRemovedStyles = html.replace(/<style[\s\S]*?<\/style>/g, '');
 
   return Utils.sanitize(truncatise(htmlRemovedStyles, {
-    TruncateLength: truncationCharCount || excerptMaxChars,
+    TruncateLength: truncationCharCount - truncationCharCount/4 || excerptMaxChars,
```

Comment:
Parenthesize, and also Math.floor this (truncationCharCount isn't guaranteed to be divisible by 4, although it happens to be with the parameters we've set.)

Reply:
```suggestion
    TruncateLength: Math.floor(truncationCharCount - truncationCharCount/4) || excerptMaxChars,
```